docs/api-spec.json

```diff
@@ -0,0 +1,442 @@
+{
+  "scriptappy": "1.0.0",
```

Comment:
What does this version mean? Is this the API spec version?

Reply:
It's scriptappy schema version..